database/migrations/2020_08_03_000000_populate_settings_table.php

```diff
@@ -0,0 +1,607 @@
+                'value' => null,
+                'created_at' => date('Y-m-d H:i:s'),
+                'updated_at' => date('Y-m-d H:i:s'),
+            ],
+            [
+                'key' => 'legacy.report.logo_file_name',
+                'value' => 'brasao.png',
+                'created_at' => date('Y-m-d H:i:s'),
+                'updated_at' => date('Y-m-d H:i:s'),
+            ],
+            [
+                'key' => 'legacy.report.show_error_details',
+                'value' => true,
+                'created_at' => date('Y-m-d H:i:s'),
+                'updated_at' => date('Y-m-d H:i:s'),
+            ],
+            [
+                'key' => 'legacy.report.default_factory',
+                'value' => 'Portabilis_Report_ReportFactoryPHPJasper',
+                'created_at' => date('Y-m-d H:i:s'),
+                'updated_at' => date('Y-m-d H:i:s'),
+            ],
+            [
+                'key' => 'legacy.report.source_path',
+                'value' => '/var/www/ieducar/ieducar/modules/Reports/ReportSources/',
+                'created_at' => date('Y-m-d H:i:s'),
+                'updated_at' => date('Y-m-d H:i:s'),
+            ],
+            [
+                'key' => 'legacy.report.diario_classe.dias_temporarios',
+                'value' => '30',
+                'created_at' => date('Y-m-d H:i:s'),
+                'updated_at' => date('Y-m-d H:i:s'),
+            ],
+            [
+                'key' => 'legacy.report.remote_factory.url',
+                'value' => null,
+                'created_at' => date('Y-m-d H:i:s'),
+                'updated_at' => date('Y-m-d H:i:s'),
+            ],
+            [
+                'key' => 'legacy.report.remote_factory.token',
+                'value' => null,
+                'created_at' => date('Y-m-d H:i:s'),
+                'updated_at' => date('Y-m-d H:i:s'),
+            ],
+            [
+                'key' => 'legacy.report.remote_factory.this_app_name',
+                'value' => null,
+                'created_at' => date('Y-m-d H:i:s'),
+                'updated_at' => date('Y-m-d H:i:s'),
+            ],
+            [
+                'key' => 'legacy.report.remote_factory.username',
+                'value' => null,
+                'created_at' => date('Y-m-d H:i:s'),
+                'updated_at' => date('Y-m-d H:i:s'),
+            ],
+            [
+                'key' => 'legacy.report.remote_factory.password',
+                'value' => null,
+                'created_at' => date('Y-m-d H:i:s'),
+                'updated_at' => date('Y-m-d H:i:s'),
+            ],
+            [
+                'key' => 'legacy.report.remote_factory.logo_name',
+                'value' => null,
+                'created_at' => date('Y-m-d H:i:s'),
+                'updated_at' => date('Y-m-d H:i:s'),
+            ],
+        ];
+        
+        DB::table('public.settings')->insert($settings);
```

Comment:
Eu sugiro simplificar, assim não precisa repetir muito as linhas e vai ficar mais fácil o CR o/

```suggestion
        $settings = [
            'legacy.app.name' => 'i-Educar',
        ];

        collect($settings)->each(function ($value, $key) {
            Setting::query()->updateOrCreate([
                'key' => $key,
            ], [
                'value' => $value,
            ]);
        });
```

Reply:
Ajuste realizado @edersoares.